libevmasm/AssemblyItem.cpp

```diff
@@ -116,15 +116,17 @@ void AssemblyItem::setPushTagSubIdAndTag(size_t _subId, size_t _tag)
 	setData(data);
 }
 
-size_t AssemblyItem::bytesRequired(size_t _addressLength, Precision _precision) const
+size_t AssemblyItem::bytesRequired(size_t _addressLength, Precision _precision, bool _usePush0) const
```

Comment:
I'd also just pass down ``EVMVersion`` directly hear instead of a ``bool``, it's in general reasonable to assume that this may depend on the evm version and it makes it clearer what the argument is for compared to just a bool that's rather opaque at call sites :-).